config/CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ the earthdata-varinfo configuration file.
 This version of the configuration file schema makes several significant changes
 to simplify the schema for broader use.
 
+### Changed:
+
+* The casing of all attributes in the configuration file schema has been
+  updated to be `PascalCase` throughout. Affected schema attributes include:
+  * `CFOverrides`
```

Comment:
Something I've been toying with through the last couple of PRs... is `CFOverrides` the best name now? Part of me thinks `MetadataOverrides` would be better, but that feels like a big change. I'd love to hear some thoughts here.

Reply:
It's only in the codebase 19 times.  It is a big change, but is it better? Probably.

Reply:
I think the main client impact is really the change in the configuration file section name.

I think the only other two user-facing places that would be impacted are the `CFConfig.cf_overrides` attribute and the `CFConfig.get_cf_overrides` method (I think everything else is just plumbing). If I'm honest, my guess is that those things aren't being called by too many end users.

Given that this is going to be the most disruptive change to the package, now is the time to do it. I think I'm leaning to making this change. (And if it seems terrible, I can always revert the commit before merging this PR)